### DISCUSSION TITLE
Documentation update

### DISCUSSION THREAD

**Discussion by givip** [simolus3/drift#1010] [Ideas]
This should be definitely added to docs: https://github.com/simolus3/moor/issues/973#issuecomment-746328513

**Comment by simolus3** [comment]
I agree! I've added [a small example](https://develop--moor.netlify.app/docs/using-sql/custom_queries/#custom-select-statements) for this to the docs on `develop`.

**Reply by givip** [reply]
Please don't forget to mention question mark syntax with numbers, like ?1, ?2, ?
